assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm

```diff
@@ -73,7 +73,8 @@ can be either all members of the site, all members of selected groups, or the se
 			<input type="radio" id="groupAssignment" name="$name_AssignTo" value="$value_AssignTo_Groups"
 				onclick="$hideGB$('#groupTableContainer').fadeIn('slow'); ASN_INE.validateGroupSelection(); ASN_INE.handleAssignToChangeForPeerAssessment(); ASN.resizeFrame(); if($!isTimesheet){ASN.disableTimesheetSetupSection();}"
 				#if ($value_AssignTo == $value_AssignTo_Groups && $!groupsList && !$selectedGroupsNotFound) checked="checked" #end
-				#if ($disableGroupAssignment) disabled="disabled" #end
+				#if ($disableGroupAssignment) disabled="disabled" #end 
+				#if ($relatedGroupsToUser.size() > 0) checked="" #end
```

Comment:
Review the variable name here:

https://github.com/mpellicer/sakai-apereo/commit/1077cc811c7edc405ede760b7cda02d8ed224ff3